test/tstRay.cpp

```diff
@@ -172,7 +172,7 @@ BOOST_AUTO_TEST_CASE(intersects_box)
   {                                                                            \
     float t0;                                                                  \
     float t1;                                                                  \
-    constexpr auto inf = KokkosExt::ArithmeticTraits::infinity<float>::value;  \
+    auto const inf = KokkosExt::ArithmeticTraits::infinity<float>::value;      \
```

Comment:
I would prefer if we can only make this `const` for `MSVC` and `constexpr` otherwise.

Reply:
I agree with @masterleinad.